lib/travis/build/git/clone.rb

```diff
@@ -63,6 +71,16 @@ def dir
           def config
             data.config
           end
+
+          def github?
+            if md = /[^@]+@(.*):/.match(data.source_url)
```

Comment:
It's a nitpick, but assigning in an `if` statement is definitely a no-no according to the ruby style guide.